examples_tests/33.Draw3DLine/main.cpp

```diff
@@ -113,7 +86,9 @@ int main()
 	matrix3x4SIMD view = matrix3x4SIMD::buildCameraLookAtMatrixRH(core::vectorSIMDf(0, 0, -10), core::vectorSIMDf(0, 0, 0), core::vectorSIMDf(0, 1, 0));
 	auto viewProj = matrix4SIMD::concatenateBFollowedByA(proj, matrix4SIMD(view));
 	core::smart_refctd_ptr<video::IGPUGraphicsPipeline> pipeline;
+	auto lines_buffer = device->createDeviceLocalGPUBufferOnDedMem(10);
```

Comment:
whats going on with the `10`?

Reply:
Oops, just was having an error so wasn't sure if 0 was a valid input, so i made it 10 "just in case" and forgot to return.